templates/slim/block_image.html.slim

```diff
@@ -1,10 +1,11 @@
-.imageblock(id=@id class=[@style,role]
-    style=[("text-align: #{attr :align}" if attr? :align),("float: #{attr :float}" if attr? :float)].compact.join('; '))
-  .content
-    - if attr? :link
-      a.image href=(attr :link)
+- unless attr(1) == 'background'
```

Comment:
Giving it some more thought, if you stick with (attr 1), then it still would be possible to provide alt text using the `alt` attribute explicitly. So now I'm inclined to say keep it how it is.